otel-extension/src/it/scala/io/scalac/mesmer/instrumentation/akka/actor/otel/AkkaActorTest.scala

```diff
@@ -196,7 +196,7 @@ final class AkkaActorTest
   }
 
   it should "record mailbox size properly" in {
-    val processingTime = 200
+    val processingTime = 1000
```

Comment:
I introduced this change too in https://github.com/ScalaConsultants/mesmer/pull/616, I think we can skip it here and keep it to a version bump only.

Reply:
@ptrdom Sure, the problem is that if I don't increase this time, the Windows build fails and won't allow me to merge the PR

Reply:
Ah, damn 🤦 @lgajowy what should we do?

Reply:
I knew that it's going to be a flaky test! ;/ Let's merge this PR and then let's fix anything downstream